src/app/components/views/generate-url/generate-url.component.ts

```diff
@@ -7,6 +8,19 @@ import { Component, OnInit } from '@angular/core';
 })
 export class GenerateUrlComponent implements OnInit {
 
+  linkdata : Link = {
+    redirectTo : "https://angular.io/tutorial/toh-pt1",
+    name : "temp",
+    customurl : "",
+    status : true,
+  };
+
+  WillbeAdded(){
+    this.linkdata.name= (<HTMLInputElement>document.getElementById("ctag")).value;
```

Comment:
Can we make use of Angular Form Group instead of writing in this way? Check [this](src/app/components/views/login/login.component.html)